deploy/helm/cht-chart/templates/couchdb-n-deployment.yaml

```diff
@@ -0,0 +1,101 @@
+{{- $root := . -}}
+{{- if .Values.couchdb.clusteredCouch_enabled }}
+{{- range $i, $e := until (int .Values.clusteredCouch.noOfCouchDBNodes) }}
+{{ $nodeNumber := add $i 1 }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    cht.service: couchdb-{{ $nodeNumber }}
+  name: cht-couchdb-{{ $nodeNumber }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      cht.service: couchdb-{{ $nodeNumber }}
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        cht.service: couchdb-{{ $nodeNumber }}
+    spec:
+      tolerations:
```

Comment:
I am curious here. There are a bunch of tolerations configured, particularly for couch db deployments, but there isn't documentation for applying the taints that make them applicable. Would you consider adding this documentation to a README (unless I missed the docs somewhere :) )?

Reply:
Good point. This will be added in the formal documentation on the docs site.